warehouse/accounts/models.py

```diff
@@ -111,6 +120,18 @@ class UnverifyReasons(enum.Enum):
     SoftBounce = "soft bounce"
 
 
+class TwoFactor(db.Model):
```

Comment:
I'm still not a huge fan of this extra model to store 20 bytes. I liked the idea you posted somewhere else of just moving ``totp_secret`` to the ``User`` model, and then when security keys get added, adding a dedicated model for those ala ``Email``.

Reply:
Alright, sounds good. I'll revert back to having `totp_secret` on the `User` model directly.